staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -92,6 +97,8 @@ type AggregateCheckResult struct {
 	// ForbiddenDetails is a slice of the forbidden details from all the forbidden checks. It may include empty strings.
 	// ForbiddenReasons and ForbiddenDetails must have the same number of elements, and the indexes are for the same check.
 	ForbiddenDetails []string
+	// ErrLists is a slice of the field errors from all the forbidden checks.
+	ErrLists []field.ErrorList
```

Comment:
Would this be more useable as a map of forbidden reason to ErrList? `map[string]field.ErrorList`? The only reason the other two are represented as parallel lists is to optimize the message formatting, but I'm not sure that applies here.

Reply:
Yes, thanks for the advice.